### PR TITLE
qiibee api reference

### PR DESCRIPTION
**Pull request by danoctavian**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -19,221 +17,620 @@ search: true
...
+  -H "Authorization: Bearer my_very_secret_api_key"
 ```
 
-```javascript
-const kittn = require('kittn');
+> Make sure to replace `my_very_secret_api_key` with your API key.
+
+qiibee uses API keys to allow access particular endpoints that are only accessible to brands. 
+
+For those particular endpoints it is expected that the the request contains the following header:
```
Comment (cupaco) [inline source/index.html.md]: remove 1x the

**Review by aaroncolaco — APPROVED** [review]

File: source/index.html.md [hunk 2]
```diff
@@ -19,221 +17,620 @@ search: true
...
+  "data": "0xa9059cbb000000000000000000000000b99c958777f024bc4ce992b2a0efb2f1f50a4dcf000000000000000000000000000000000000000000000000000000000000000a",
+  "chainId": 17224
+}
 ```
 
-This endpoint retrieves all kittens.
+This endpoint returns a raw transaction (unsigned) which uses the nonce based on the blockchain `transactionCount` for that address.
+
+
+<aside class="warning"> If an SDK is available for your programming language, we highly recommend you use the `Wallet` class to construct valid transactions and send them instead of this function, since it takes care of transaction construction and signing for you. </aside>
```
Comment (cupaco) [inline source/index.html.md]: we highly recommend you *to* use the?

File: source/index.html.md [hunk 3]
```diff
@@ -19,221 +17,620 @@ search: true
...
 
-api = Kittn::APIClient.authorize!('meowmeowmeow')
-api.kittens.get(2)
+signed_tx_string = '0xf8aa8305ee6e80830f424094f80c1e40d50f383330a886f03f147f49342ad7c780b844a9059cbb00000000000000000000000087265a62c60247f862b9149423061b36b460f4bb000000000000000000000000000000000000000000000000000000000000000a8286b5a0e8a236e4b57e79ee0eec88d8bc7a7ef1c44a1d181bad366621e72c341490e550a0683ce6443acc4384ea2b18fcaba416b8f7379963b6a0c596cea423cade502eb3'
+
+api.post_transaction(signed_tx_string)
 ```
 
+```shell
+curl -d '{"data": "0xf8aa8305ee6e80830f424094f80c1e40d50f383330a886f03f147f49342ad7c780b844a9059cbb00000000000000000000000087265a62c60247f862b9149423061b36b460f4bb000000000000000000000000000000000000000000000000000000000000000a8286b5a0e8a236e4b57e79ee0eec88d8bc7a7ef1c44a1d181bad366621e72c341490e550a0683ce6443acc4384ea2b18fcaba416b8f7379963b6a0c596cea423cade502eb3" }' -H "Content-Type: application/json" -X POST  https://apitesting.qiibee.com/transactions
```
Comment (cupaco) [inline source/index.html.md]: why are we mixing api and apitesting in the examples?

File: source/index.html.md [hunk 4]
```diff
@@ -19,221 +17,620 @@ search: true
...
+
+```json
+{
+  "CHF": "0.0210492399",
+  "EUR": "0.0209613126"
 }
 ```
 
-This endpoint deletes a specific kitten.
+Returns the FIAT price of one unit of a given Loyalty Token. This endpoint uses a third-party provider to get the ETH echange rate. The QBX/ETH Exchange rate is fetched from the Coinsuper exchange.
```
Comment (cupaco) [inline source/index.html.md]: e*x*change rate

File: source/index.html.md [hunk 5]
```diff
@@ -19,221 +17,620 @@ search: true
...
+Parameter | Default | Description
+--------- | ------- | -----------
+from | None | Loyalty Token contract address.
+to | USD | Comma separated currency symbols list to convert into. For example: USD,EUR,CHF. 
+
+
+## Get price history
+
+```shell
+curl "https://api.qiibee.com/prices/history?from= 0x883ed48b3210082cf82fb92ce81f0b17bdec4f81&to=AUD&limit=3"
```
Comment (cupaco) [inline source/index.html.md]: from=<space>, remove space in url

File: source/index.html.md (hunk 3 above)
Comment (danoctavian) [reply source/index.html.md]: paste error

File: source/index.html.md (hunk 2 above)
Comment (danoctavian) [reply source/index.html.md]: yeah

**Review by cupaco — APPROVED** [review]

**Pull request merged**